logstash-output-elasticsearch.gemspec

```diff
@@ -31,7 +31,7 @@ Gem::Specification.new do |s|
 
   if RUBY_PLATFORM == 'java'
     s.platform = RUBY_PLATFORM
-    s.add_runtime_dependency "manticore", '>= 0.5.2', '< 1.0.0'
+    s.add_runtime_dependency "manticore", '>= 0.5.4', '< 1.0.0'
```

Comment:
should work: https://github.com/logstash-plugins/logstash-mixin-http_client/blob/master/logstash-mixin-http_client.gemspec#L21